src/lsp/swift-mesonlsp.ts

```diff
@@ -0,0 +1,64 @@
+import * as os from "os";
+
+import {
+  LanguageClient,
+  LanguageClientOptions,
+  ServerOptions,
+  Executable,
+  TransportKind
+} from "vscode-languageclient/node";
+import {
+  ExtensionContext,
+} from "vscode"
+import { LanguageServerClient } from "../lsp";
+
+export class SwiftMesonLspLanguageClient extends LanguageServerClient {
+  ls: LanguageClient | null = null
```

Comment:
I don't think is necessary? Looks like you already have one in the parent class.